### PR TITLE
chore(update): replaced stand-alone with standalone

### LINKED ISSUE #9599: docs: replace `stand-alone` with `standalone`
### Current State

Mixed use of two spellings at the moment.

### Desired State

Should stick to `standalone`. Checked references: Microsoft docs, Google docs.

### PR DESCRIPTION
**Pull request by mau211**
### Description

Updated text found as stand-alone and replaced with `standalone`

Fixes #9599 

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (kayx23) [inline package-lock.json:1]: please remove `package-lock.json` file

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -1302,7 +1302,7 @@ title: CHANGELOG
 
 ### Core
 
-- :sunrise: **[增加单机模式](https://github.com/apache/incubator-apisix/blob/master/docs/zh/latest//stand-alone-cn.md)**: 使用 yaml 配置文件来更新 APISIX 的配置，这对于 kubernetes 更加友好。 [#464](https://github.com/apache/incubator-apisix/pull/464)
+- :sunrise: **[增加单机模式](https://github.com/apache/incubator-apisix/blob/master/docs/zh/latest//standalone-cn.md)**: 使用 yaml 配置文件来更新 APISIX 的配置，这对于 kubernetes 更加友好。 [#464](https://github.com/apache/incubator-apisix/pull/464)
```
Comment (kayx23) [inline docs/zh/latest/CHANGELOG.md:1305]: This page seems to not exist either way. You can revert the change for this one.

File: docs/zh/latest/README.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ A/B 测试、金丝雀发布（灰度发布）、蓝绿部署、限流限速、
     - [控制台](https://github.com/apache/apisix-dashboard): 操作 APISIX 集群。
     - 版本控制：支持操作的多次回滚。
     - CLI：使用命令行来启动、关闭和重启 APISIX。
-    - [单机模式](stand-alone.md)：支持从本地配置文件中加载路由规则，在 kubernetes(k8s) 等环境下更友好。
+    - [单机模式](standalone.md)：支持从本地配置文件中加载路由规则，在 kubernetes(k8s) 等环境下更友好。
```
Comment (kayx23) [inline docs/zh/latest/README.md:141]: Same with this one. The file seems to be gone.

File: docs/en/latest/deployment-modes.md [hunk 4]
```diff
@@ -159,7 +159,7 @@ deployment:
 
 ## Standalone
 
-Turning on the APISIX node in Stand-alone mode will no longer use the default etcd as the configuration center.
+Turning on the APISIX node in standalone mode will no longer use the default etcd as the configuration center.
```
[the remark this answers is not in the document]
Comment (mau211) [reply docs/en/latest/deployment-modes.md]: Update

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]
